go/heif/heif.go

```diff
@@ -827,6 +827,63 @@ func (img *Image) GetImage() (image.Image, error) {
 					},
 				},
 			}
+		case ChromaInterleavedRRGGBB_BE:
+			rgb, err := img.GetPlane(ChannelInterleaved)
+			if err != nil {
+				return nil, err
+			}
+			width := img.GetWidth(ChannelInterleaved)
+			height := img.GetHeight(ChannelInterleaved)
+			rgba := make([]byte, width*height*8)
+			read_pos := 0
+			write_pos := 0
+			stride_add := rgb.Stride - width*6
+			for y := 0; y < height; y++ {
+				for x := 0; x < width; x++ {
+					rgba[write_pos] = rgb.Plane[read_pos]
+					rgba[write_pos+1] = rgb.Plane[read_pos+1]
+					rgba[write_pos+2] = rgb.Plane[read_pos+2]
+					rgba[write_pos+3] = rgb.Plane[read_pos+3]
+					rgba[write_pos+4] = rgb.Plane[read_pos+4]
+					rgba[write_pos+5] = rgb.Plane[read_pos+5]
+					read_pos += 6
```

Comment:
Shouldn't the Go image alpha channel filled with an opaque value here?

Doesn't have to image data to be shifted left into the MSB bits for Go? Libheif fills the n-bit value into the least significant n bits. So it might have to be shifted left first.
out = (in << (16-n)) | (in >> (2*n-16))